src/lib/provable/int.ts

```diff
@@ -1142,14 +1146,27 @@ class Int64 extends CircuitValue implements BalanceChange {
   }
 
   /**
-   * Negates the value.
-   *
-   * `Int64.from(5).neg()` will turn into `Int64.from(-5)`
+   * @deprecated Use {@link negV2()} instead.
```

Comment:
should we add this part to the changelog as well?

Reply:
ok we can do that